.github/workflows/publish-doc-to-remote.yml

```diff
@@ -15,6 +15,9 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: '3.8'
+    - name: Install sndfile library
+      run: |
+        sudo apt-get install libsndfile1-dev
```

Comment:
Note sure if we should add this dependency. Does librosa complains on other OS than Linux?

And if we do add a dependency, I would add it in a file where all deps are listed and then have the CI read that file.

Reply:
It did not complain on my Macbook locally when running the music example, but it does not mean that there is no dependency that I resolved sooner by installing something via `homebrew`. 
I will try running this GH Actions on `macos-latest` and `windows-latest` and see how it ends up ! What is sure is that it complains on `ubuntu-latest` and installing just the python library `librosa` is not enough (see [here](https://github.com/deepcharles/ruptures/runs/1830444747?check_suite_focus=true)).
The script `.github/workflows/check-docs.yml` is also impacted. 

Depending on this, we will see how to handle dependencies ! The `libsndfile1-dev` dependency also appear in the `.binder/apt.txt` so that when Binder build the image, it has everything it needs !

Reply:
Seems to work on `macos-latest` !
I will make the change on `.github/workflows/publish-doc-to-remote.yml` as well !

Last thing, but for me it is fine since that is for Binder, `libsndfile1-dev` still appears in Binder config file `.binder/apt.txt` ! WDYT on this particular one ? 
Not sure, but if we are firmly against having this dependency on the `ruptures` repo, we might be able to build our own Docker image and just specify it using Binder [advance configuration](https://mybinder.readthedocs.io/en/latest/tutorials/dockerfile.html). Seems a bit far fetch though !